src/modules/bspwm.cpp

```diff
@@ -222,6 +243,14 @@ namespace modules {
       return false;
     }
 
+    auto index{1};
+
+    for (auto p : positions) {
+      if (p == curr_pos)
+        break;
```

Comment:
```suggestion
      if (p == curr_pos) {
        break;
      }
```

Also, `curr_pos` may not be initialized here.